tmt/steps/discover/shell.py

```diff
@@ -145,7 +145,8 @@ def from_serialized(cls, serialized: Dict[str, Any]) -> 'TestDescription':
         obj = super().from_serialized(serialized)
         obj.link = tmt.base.Links(serialized['link'])
         obj.require = [
-            require if isinstance(require, str) else tmt.base.RequireFmfId.from_serialized(require)
+            tmt.base.RequireSimple.from_spec(require)
+            if isinstance(require, str) else tmt.base.RequireFmfId.from_serialized(require)
```

Comment:
I guess we need to cover `recommend` here as well? And update `to_spec()`, `to_serialized()` in a similar way?

Reply:
Hopefully fixed in [8bbfb89](https://github.com/teemtee/tmt/pull/1630/commits/8bbfb893ec88d5e5da6e7af39cd3e851fa04d03b)

Reply:
Yes, looks good.